### DISCUSSION TITLE
Cannot view CircleCI failures

### DISCUSSION THREAD

**Discussion by lyndsey-ferguson** [fastlane/fastlane#16834] [Q&A] [answered]
Hello,

I have a [PR 16833](https://github.com/fastlane/fastlane/pull/16833) that is fixing an issue in Scan for log collection from Simulators. I cannot view the CircleCI failures because I would have to authorize CircleCI to view my employer's private repos. 

I cannot do that. Can someone look at the failures and let me know what is failing?

Thanks! 🚀

**Comment by joshdholtz** [comment, marked as answer]
I tried viewing on my phone for you but it turns out that CircleCI doesn’t give me a great mobile view. And this might be a good reason to switch to GitHub Actions now 😇 But I will send you those on that PR later to tonight!

Do the tests not fail locally for you?

**Reply by lyndsey-ferguson** [reply]
I ran the `bundle exec rspec` and the other rubocop command, but am running them again locally in case I broke something.

**Reply by lyndsey-ferguson** [reply]
Um, I must have reverted something when rebasing, I found a failing test.

**Reply by lyndsey-ferguson** [reply]
Okay, I found it, thanks for responding so quickly @joshdholtz; I'll post in Slack for a code-review/approval/merge?
